src/Networks/Bitcoin/Blockcore.Networks.Bitcoin/BitcoinMain.cs

```diff
@@ -66,8 +67,15 @@ public BitcoinMain()
                 [BitcoinBIP9Deployments.Segwit] = new BIP9DeploymentsParameters("Segwit", 1, 1479168000, 1510704000, BIP9DeploymentsParameters.DefaultMainnetThreshold)
             };
 
+            ConsensusProtocol consensusProtocol = new ConsensusProtocol()
+            {
+                ProtocolVersion = ProtocolVersion.WITNESS_VERSION,
+                MinProtocolVersion = ProtocolVersion.SENDHEADERS_VERSION,
```

Comment:
Why is the MinProtocolVersion different than NodeSettings MinProtocolVersion in the Bitcoin CLI project?

Reply:
in reality `WITNESS_VERSION` and `SENDHEADERS_VERSION` are the same number (both at 70012) we should get rid of the wtiness one though (another PR)